.github/styles/README.md

```diff
@@ -0,0 +1,24 @@
+# Style Guide
+The team follows the Google developer's and the Microsoft's documentation style guide. The style rules are configured with the help of Vale linter.
+
+## Prerequisites
+Please ensure that the below prerequisites are met before you configure vale linter. You should have:
+
+* [Brew](https://brew.sh) installed on your mac/window.
+* Vale installed on your mac/window. 
```

Comment:
```suggestion
* Vale installed on your Mac/Windows machine.
```

Reply:
We don't really need to use `machine`. Mac/window is understandable.

Reply:
It doesn't really matter here since this content is for internal use / not for users, but we should probably be careful in other docs that we use "Windows" instead of "window"